src/components/EventList.jsx

```diff
@@ -7,7 +7,11 @@ const Event = props => (
   <div>
     <p className="mb-0">{props.event.dateLabel}</p>
     <p>
-      <span className="accent"> {props.event.title}: </span>{" "}
+      <span className="accent">
+        {" "}
+        {props.event.title}
+        {props.event.description === "" ? ": " : ""}{" "}
```

Comment:
Ooo I like this, did notice some dangling : when no description

Reply:
haha, it was bothering me a bunch 😆